src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -30,7 +32,8 @@
             + PREFIX_SORT_DIRECTION + "desc";
 
     public static final String MESSAGE_SUCCESS = "Successfully sorted";
-    public static final String MESSAGE_INVALID_ATTRIBUTE = "This attribute is invalid";
+    public static final String MESSAGE_INVALID_ATTRIBUTE = "This attribute is invalid.\nThe available attributes are"
+            + " empty (default), numPolicies, premium and lastContacted.";
```

Comment:
Minor nitpick but maybe you could wrap the enum values with an apostrophe or backtick to make it clear what the possible inputs are?

Reply:
Good suggestion will do this!